pylintrc

```diff
@@ -0,0 +1,3 @@
+[master]
+disable=
```

Comment:
This discards the pylint error about the line being too long. Unfortunately, the file would be too many lines, if I broke up all of the urls/uri's.